FormalConjectures/ErdosProblems/386.lean

```diff
@@ -0,0 +1,64 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Erdős Problem 386
+
+*References:*
+- [erdosproblems.com/386](https://www.erdosproblems.com/386)
+-/
+
+namespace Erdos386
+
+/-
+This is the structure of the equation:
+-/
+
+def erdos_386_equation (a b : ℕ) (A : Finset ℕ) :=
+  Nat.choose a b = ∏ α ∈ A, α
+
+/-
+The following set contains all solutions `(n, k, P)` to the Erdős problem 386.
+A solution is a tuple `(n, k, P)`, where `n ≥ 2` and `k` are an integers and `P`
+is a non-empty finite set of
+distinct prime numbers, such that it's product is binomial coefficient n choose k.
+-/
+def erdos_386_solutions : Set (ℕ × ℕ × Finset ℕ) := {
+  (n, k, P) |
+    (n ≥ 2 ∧ 2 ≤ k ∧ k ≤ n - 2) ∧
```

Comment:
```suggestion
    (n ≥ 2 ∧ 2 ≤ k ∧ k ≤ n - 2) ∧
```
I would put either `n ≥ 4` or no condition on `n` here, since for `n < 3` no such `k` exist.